docs/design/Non_Admin_Controller_design.md

```diff
@@ -0,0 +1,83 @@
+# Non-Admin Backup/Restore Design
+
+## Background
+OADP (Openshift API for Data Protection) Operator currently requires cluster admin access for performing Backup and Restore operations of applications deployed on the OpenShift platform. This design intends to enable the ability to perform Backup and Restore operations of their own application namespace for namespace owners aka non-admin users.
```

Comment:
Usually in docs the first mention of an acronym spells it out. I don't know if that's needed here since this is a design *within* the OADP project, though.

Reply:
```suggestion
OADP (Openshift API for Data Protection) Operator currently requires cluster admin access for performing Backup and Restore operations of applications deployed on the OpenShift platform. This design intends to enable non-admin users the ability to perform Backup and Restore operations of their application namespace.
```